### DISCUSSION TITLE
Marquee

### DISCUSSION THREAD

**Discussion by daudln** [shadcn-ui/ui#2682] [Ideas]
It will be nicer to have Marquee component ✨

- Features ✨

1. Reverse mode.

2. Fade mode.

3. Pause on hover.

4. Vertical mode.

I am motivated with this tweet!

https://twitter.com/selemondev/status/1754264848123273488?t=AF2whC8fEF4-jvA5_CmFkw&s=19

**Comment by wilsonhou** [comment]
https://github.com/justin-chu/react-fast-marquee

react-fast-marquee is a bit older but seems to work quite well. could be suitable here.

**Reply by daudln** [reply]
Suitable and perfect. May it will be nicer to have it here @shadcn

**Reply by johanguse** [reply]
I am not sure if this is 100% compatible with NextJS. See the[ issue 100](https://github.com/justin-chu/react-fast-marquee/issues/100)

I've used this with no problem: https://github.com/devnomic/marquee
